ui/components/app/transaction-detail/transaction-detail.component.js

```diff
@@ -17,7 +17,11 @@ const GasLevelIconMap = {
   custom: '⚙',
```

Comment:
The changes in this file are no longer needed because we hide the edit button now https://github.com/MetaMask/metamask-extension/pull/12576/files#diff-a6209b43f1af7f5cf53ecdaba18ee1e22d0bdc774bc369daaefa4583a9d25fb2R573-R575